bliss/simulator/decoder.py

```diff
@@ -143,7 +142,10 @@ def render_images(self, tile_cat: TileCatalog, image_ids):
                     # essentially all the runtime of the simulator is incurred by this call
                     # to drawImage
                     galsim_obj.drawImage(
-                        offset=offset, method="auto", add_to_image=True, image=gs_img
+                        offset=offset,
+                        method=self.psf_draw_method,
```

Comment:
Would you add a comment somewhere explaining why we have to use a different psf draw method for DES?

Reply:
It's currently in `bliss/surveys/des.py:357-359`—should it be moved elsewhere?